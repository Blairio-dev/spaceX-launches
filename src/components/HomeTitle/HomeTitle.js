import React from "react";
import styled from "@emotion/styled";
import { Image } from "../../components";
import spaceXLogo from "../../assets/img/spacex-logo.png";
import { breakpoints, colours } from "../../assets/tokens";

const StyledTitle = styled("h1")`
  color: ${colours.text};
  font-family: "Brandon Grotesque Regular";
  font-size: 20px;
  font-weight: 400;
  line-height: 1;
  margin: 0 0 0 -12px;
`;

const StyledWrapper = styled("div")`
  align-items: baseline;
  display: flex;
  height: fit-content;

  @media (max-width: ${breakpoints.sm}) {
    justify-self: flex-start;
    margin-bottom: 24px;
    width: 100%;
  }
`;

const HomeTitle = () => (
  <StyledWrapper>
    <Image
      alt="SpaceX Logo"
      height="22px"
      source={spaceXLogo}
      key="spacex-logo"
    />
    <StyledTitle>LAUNCHES</StyledTitle>
  </StyledWrapper>
);

HomeTitle.propTypes = {};

export { HomeTitle };
