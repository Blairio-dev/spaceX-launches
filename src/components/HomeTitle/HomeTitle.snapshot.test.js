import React from "react";
import renderer from "react-test-renderer";
import { HomeTitle } from "./HomeTitle";

const DEFAULT_PROPS = {};

describe("HomeTitle Snapshots", () => {
  it("Renders with default props", () => {
    const tree = renderer.create(<HomeTitle {...DEFAULT_PROPS} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
