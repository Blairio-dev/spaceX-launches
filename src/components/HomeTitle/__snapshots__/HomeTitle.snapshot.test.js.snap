// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeTitle Snapshots Renders with default props 1`] = `
<div
  className="css-197d2zo"
>
  <img
    alt="SpaceX Logo"
    height="22px"
    src="spacex-logo.png"
  />
  <h1
    className="css-n8q6wh"
  >
    LAUNCHES
  </h1>
</div>
`;
